components/sections/Projects/ProjectTitle.jsx

```diff
@@ -1,16 +1,27 @@
 import Image from "next/image";
 
 const ProjectTitle = ({ name, openProjectPage, width, active, index }) => {
+  const MINIMUM_MOBILE_WIDTH = 1024;
```

Comment:
This is not minimum mobile width currently, it's the maximum width for all small screens.
It can be named instead `MAXIMUN_SMALL_BREAKPOINTS_WIDTH` or something else related to that.